core/lib/zksync_core/src/consensus/storage/mod.rs

```diff
@@ -255,14 +255,22 @@ impl BlockStore {
             .await
             .wrap("payload()")?
             .context("miniblock disappeared")?;
-        let (genesis, _) = zksync_consensus_bft::testonly::make_genesis(
-            &[validator_key.clone()],
-            payload.encode(),
-            number,
-        );
-        txn.insert_certificate(ctx, &genesis.justification, self.inner.operator_address)
-            .await
-            .wrap("insert_certificate()")?;
+        let mut genesis = validator::testonly::GenesisSetup {
```

Comment:
Side comment: using `testonly` outside of tests looks a bit odd.

Reply:
done